osu.Game/Overlays/Profile/Header/Components/GroupBadge.cs

```diff
@@ -35,6 +35,11 @@ public GroupBadge(APIUserGroup group)
             CornerRadius = 8;
 
             TooltipText = group.Name;
+
+            if (group.IsProbationary)
+            {
+                Alpha = 0.6f;
```

Comment:
What's going on here??

Reply:
probationary groups badges are 60% transparent, while using the bgColor at 100% opacity, but normal groups have bgColor at 75% opacity but the badge itself is not transparent, just the backgrounds.

Reply:
This seems really confusing and verging on "this is implemented incorrectly at osu-web end" if you're copying from there.

Either way, it needs inline explanation and linking to the source it was copied from if it's going to exist.
;

Reply:
is this comment good enough?, im not sure what exactly im supposed to specify
```
// Normal badges background opacity is 75%, probationary is full opacity as the whole badge gets a bit transparent
// Goal is to match osu-web so this is the most accurate it can be, its a bit scuffed but it is what it is
// Source: https://github.com/ppy/osu-web/blob/master/resources/css/bem/user-group-badge.less#L50
```

Reply:
Yep, that sounds fine.

Reply:
pushed